### PR TITLE
Feature/json env

### PR DESCRIPTION
**Pull request by qqmbr4k**
Added support for ENV VARS links in JSON

### CODE REVIEW

File: rest-server/core/src/main/scala/com/actionml/core/validate/JsonSupport.scala [hunk 1]
```diff
@@ -123,4 +128,12 @@ trait JsonSupport extends LazyLogging {
   }
 
 
+  private val env = s"system.env.(.*)".r
+  private def enrichViaSystemEnv: String => String = {
+    case env(value) => sys.env.getOrElse(value, {
+      logger.warn(s"No ENV VAR for $value")
+      value
+    })
```
Comment (andreyshikov) [inline rest-server/core/src/main/scala/com/actionml/core/validate/JsonSupport.scala:137]: @pferrel this ^^^ means silence (just warning in the logs), is it ok?

Comment (pferrel) [reply rest-server/core/src/main/scala/com/actionml/core/validate/JsonSupport.scala:137]: Sounds good.

**Review by pferrel — APPROVED** [review]
As I read this, there is no option for not enrichingViaSystemEnv. But we know the only valid use is for the engine JSON. I guess that is ok, there might be some as yet unforeseen use for Events or other JSON request bodies.

**Comment by andreyshikov** [comment]
> As I read this, there is no option for not enrichingViaSystemEnv. But we know the only valid use is for the engine JSON. I guess that is ok, there might be some as yet unforeseen use for Events or other JSON request bodies.

@pferrel e.g. I can add a param (with default to "do not use system.env") to parseAndValidate to turn on "system.env parsing"

File: Makefile [hunk 2]
```diff
@@ -34,7 +34,7 @@ clean: sbt
 		$(SBT) ++$(SCALA_VERSION) -batch server/clean
 	find $(RESTSRV_DIR) -name "__pycache__" -exec rm -r {} \; || true
 
-dist: clean clean-dist build wheel
```
Comment (andreyshikov) [inline Makefile]: @pferrel that fixes the Makefile

**Pull request merged**